package.json

```diff
@@ -13,41 +13,41 @@
   },
   "scripts": {
     "codegen": "turbo run codegen",
-    "dev": "concurrently \"yarn:dev:backend\" \"yarn:dev:frontend\"",
-    "dev:backend": "concurrently \"yarn:dev:backend:*\"",
-    "dev:backend:api": "yarn workspace @apps/hash-api dev",
-    "dev:backend:realtime": "yarn workspace @apps/hash-realtime dev",
-    "dev:backend:search-loader": "yarn workspace @apps/hash-search-loader dev",
-    "dev:frontend": "yarn workspace @apps/hash-frontend dev",
-    "external-services": "turbo run deploy --filter='@apps/hash-external-services' --",
-    "external-services:offline": "turbo run deploy:offline --filter='@apps/hash-external-services' --",
-    "external-services:test": "turbo run deploy:test --filter='@apps/hash-external-services' --",
+    "dev": "turbo run dev --filter '@apps/hash-api' --filter '@apps/hash-search-loader' --filter '@apps/hash-realtime' --filter '@apps/hash-frontend' --",
+    "dev:backend": "turbo run dev --filter '@apps/hash-api' --filter '@apps/hash-realtime' --filter '@apps/hash-search-loader' --",
+    "dev:backend:api": "turbo run dev --filter '@apps/hash-api' --",
+    "dev:backend:realtime": "turbo run dev --filter '@apps/hash-realtime' --",
+    "dev:backend:search-loader": "turbo run dev --filter '@apps/hash-search-loader' --",
```

Comment:
Do you think we should add `"persistent": true,` entries in these projects' `turbo.json` for their `dev` tasks to disallow other rules being dependent on them? The flags makes it so other tasks can't be dependent on the `dev` task (since they may run indefinitely)

Reply:
The global `turbo.json` contains `persistent: true` for `dev` because of this reason 🙂